commands/help.py

```diff
@@ -21,7 +22,7 @@ def help(update, context):
     context.bot.send_message(
         chat_id=chat_id,
         text=help_text,
-        parse_mode=ParseMode.HTML,
+        parse_mode=ParseMode.MARKDOWN,
```

Comment:
Any reason why do we need to change this back to `MARKDOWN`?

Reply:
oh sorry I had just pasted my previous code which was before the last commit change today..